tmt/utils/__init__.py

```diff
@@ -1985,16 +1988,16 @@ def read(self, path: Path, level: int = 2) -> str:
             path = self.workdir / path
         self.debug(f"Read file '{path}'.", level=level)
         try:
-            with open(path, encoding='utf-8', errors='replace') as data:
-                return data.read()
+            return path.read_text(encoding='utf-8', errors='replace')
+
         except OSError as error:
-            raise FileError(f"Failed to read '{path}'.\n{error}")
+            raise FileError("Failed to read from '{path}'.") from error
```

Comment:
Missing f-string here?

Reply:
Yeah, it's missing :/ I fixed the same problem in `write()`, and forgot to check `read()`. Fixed in 90500c70.